src/System.Text.Formatting/System/Text/Parsing/InvariantParser_ulong.cs

```diff
@@ -46,5 +46,41 @@ public static bool TryParse(Utf8String text, out ulong value, out int bytesConsu
 
             return true;
         }
+		public static bool TryParse(byte[] text, out ulong value, out int bytesConsumed)
+        {
```

Comment:
There needs to be an int index parameter as well, otherwise you can only parse integers at the beginning of the array, which is not very useful.